src/app/edited-word-entry/edited-word-entry.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Input } from '@angular/core';
+import { Output, EventEmitter } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { LanguagePair } from '../language-pair.model';
+import { WordEntry } from '../word-entry.model';
+
+@Component({
+    selector: 'app-edited-word-entry',
```

Comment:
```suggestion
    selector: 'edit-word-entry',
```

There is no need to have `app` prefix here.

Reply:
Foxed!